docs/azure_jumpstart_ag/contoso_supermarket/arc_defender_servers/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+type: docs
+weight: 100
+toc_hide: true
+---
+
+# Infrastructure security with Microsoft Defender for Servers
+
+## Overview
+
+Contoso Supermarket is aware that as they navigate the complex landscape of edge computing, it is essential to protect their servers against threats while maintaining a comprehensive view of their security posture. Microsoft Defender for Servers emerges as a vital component, playing a pivotal role in protecting servers from threats and providing unparalleled visibility into their security landscape.
+
+Microsoft Defender for Servers ensures that Contoso Supermarket's Azure Arc-enabled servers remain resilient against a wide range of cyber threats. With advanced threat protection, machine learning algorithms, and real-time security intelligence, Microsoft Defender for Servers proactively detects and neutralizes potential risks.
+
+On the other hand, maintaining visibility into the security posture of Azure Arc-enabled servers is crucial for Contoso Supermarket. Microsoft Defender for Servers offers comprehensive insights into the servers' security status, providing a centralized view of threats, vulnerabilities, and security events. By analyzing telemetry data, logs, and behavioral patterns, Contoso Supermarket gains a deep understanding of their security landscape, enabling proactive risk mitigation.
+
+## Instructions
+
+To get all of the Microsoft Defender for Servers protections, you will need to enable the Microsoft Defender for Servers plan at subscription level. Follow these steps to enable the plan in the case it is not already enabled:
+
+- Login to AZ CLI using the ```az login``` command.
+
+- Ensure that you have selected the correct subscription by using the ```az account list --query "[?isDefault]"``` command. If you need to adjust the active subscription used by Az CLI, follow [this guidance](https://docs.microsoft.com/cli/azure/manage-azure-subscriptions-azure-cli#change-the-active-subscription).
+
+- Run the following command to check whether Microsoft Defender for Servers is already enabled in your subscription or not:
+
+    ```shell
+    az security pricing show -n VirtualMachines
+    ```
+
+    ![Screenshot of checking microsoft defender for servers enabled at subscription level](./img/01.png)
+
+    > **NOTE: Proceed with the next step if the value you have got for _pricingTier_ is equals to _Free_. If the value was _Standard_, Microsoft Defender for Servers is already enabled and no additional action is required.**
+
+- Run the following command to enable Microsoft Defender for Servers in your subscription:
+
+    ```shell
+    az security pricing create -n VirtualMachines --tier 'standard'
+    ```
+
+    ![Screenshot of enabling microsoft defender for servers  at subscription level](./img/02.png)
+
+    > **NOTE: Remember to follow the steps provided in the Cleanup section to disable Microsoft Defender for Servers.**
```

Comment:
```suggestion
    > **NOTE: Remember to follow the steps provided in the Cleanup section to disable Microsoft Defender for Servers, otherwise you may incur additional charges in your subscription.**
```